docs/workflows/outputs/twilio.md

```diff
@@ -14,18 +14,20 @@ The Twilio node has a large number of configuration properties - everything need
 
 The most important part of the configuration is the Account SID and Auth Token for the Twilio account that should be used to send the SMS messages. In the example above, the node is configured to use the the Account SID `YOUR_TWILIO_ACCOUNT_SID_HERE` and the Auth Token `YOUR_TWILIO_AUTH_TOKEN_HERE`.
 
-Optionally, the maximum allowed price of an SMS message can be configured (see Twilio's [API Docs](https://www.twilio.com/docs/api/rest/sending-messages) for more information on this field).  The max price field supports payload templates, and so can be dynamically configured.
+Optionally, the maximum allowed price of an SMS message can be configured (see Twilio's [API Docs](https://www.twilio.com/docs/api/rest/sending-messages) for more information on this field).
+
+All of the account configuration inputs, including the maximum price, support [string templates](/workflows/accessing-payload-data/#string-templates).
```

Comment:
so both the old and the new docs were wrong here, the max price field does not currently support templates for some reason.